docs/developer/advanced/sharing-saved-objects.asciidoc

```diff
@@ -28,6 +28,12 @@ Ideally, most types of objects in {kib} will eventually be _shareable_; however,
 <<sharing-saved-objects-faq-share-capable-vs-shareable,_share-capable_ objects>> as a stepping stone for plugin developers to fully support
 this feature.
 
+Implementing a shareable saved object type is done in two phases:
+
+- **Phase 1**: Convert an existing isolated object type into a share-capable one. Keep reading!
+- **Phase 2**: Switch an existing share-capable object type into a shareable one, _or_ create a new shareable object type. Jump to the
```

Comment:
Does Phase 2 need to know about the Breaking changes?

Reply:
Good question -- no, these breaking changes only affect existing object types that were converted in the 8.0 release.